docs/readme.md

```diff
@@ -156,4 +156,12 @@ If the underlying data source does not allow such a fine-grained distinction, us
 		currency: 'EUR' // ISO 4217 code, required
 	}
 }
-```
\ No newline at end of file
+```
+
+## modes
+
+- `walking`
+- `train`
+- `bus` - local traffic
+- `coach` - long-distance traffic
```

Comment:
What is the distinction between local traffic and long-distance traffic, especially in other countries?